src/_guides/language/language-tour.md

```diff
@@ -3252,15 +3255,26 @@ abstract class Musical {
 }
 {% endprettify %}
 
-<div class="alert alert-info" markdown="1">
-  **Note:** Some restrictions on mixins are being removed. For details, see the
-  [proposed mixin specification.][Dart 2.0 Mixins]
+To specify that only certain types can use the mixin — for example,
+so your mixin can invoke a method that it doesn't define —
```

Comment:
Use `&mdash;` instead?

Reply:
I'm not crazy about `&mdash;`... it's not very readable.